Multisig/Data/Services/SafeTransactionService.swift

```diff
@@ -111,22 +111,29 @@ struct SafeBalancesRequest: JSONRequest {
 }
 
 struct TransactionsRequest: JSONRequest {
-    let address: String
-    let limit: Int
-    let offset: Int
+    let address: String?
+    let limit: Int?
+    let offset: Int?
     var httpMethod: String { "GET" }
-    var urlPath: String { "/api/v1/safes/\(address)/all-transactions/" }
+    var urlPath: String {
+        guard let address = address else { return "" }
```

Comment:
how about taking it from 'url'? There there will be no need to introduce 'url' in many other places.

Reply:
I like it. But that means we need to parse the URL itself, which is not desired

Reply:
the way would be to URL -> URLComponents -> use in the "urlPath" and in "query"

Reply:
But this means we need to know how to split the url, which I think we should avoid to count on